monkey/monkey_island/cc/ui/src/components/report-components/SecurityReport.js

```diff
@@ -226,6 +226,18 @@ class ReportPageComponent extends AuthComponent {
     return Object.keys(this.state.report).length === 0;
   }
 
+  getMonkeyDuration() {
+    if (this.state.report.overview.monkey_duration !== 0) {
+      return <>
+        After <span
+        className='badge badge-info'>{this.state.report.overview.monkey_duration}</span>, all monkeys finished
+        propagation attempts.
```

Comment:
This says "all monkeys finished propagation attempts." but they didn't. This is show with the latest agent stop time, even though agents are still running.